src/models/expense.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const { sequelize } = require('../utils/db');
+const { DataTypes } = require('sequelize');
+
+const Expense = sequelize.define('Expenses', {
```

Comment:
imo better to specify it directly

Reply:
thank you for approve, i didn't add "id", because when i developed on django it was not necessary, "id" is default field. But i will fix this in current task